model_analyzer/config/generate/quick_run_config_generator.py

```diff
@@ -321,6 +321,17 @@ def _get_next_model_run_config(
         model_run_config = ModelRunConfig(model_name, model_config,
                                           perf_analyzer_config)
 
+        bls_composing_model_configs = []
+        for bls_composing_model in self._bls_composing_models:
+            bls_composing_model_config = self._get_next_model_config(
+                bls_composing_model, model_index)
+            model_index += 1
```

Comment:
This model index code seems very fragile. A lot of baked in knowledge to know the implicit locations of the models. Let's make sure that is part of the refactor.

Reply:
ok, we can revisit it when refactoring